package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/SocketHandler.kt

```diff
@@ -0,0 +1,32 @@
+package cloud.eppo.android.sdkrelay
+
+import io.socket.client.IO
+import io.socket.client.Socket
+import java.net.URISyntaxException
+
+object SocketHandler {
+
+  lateinit var mSocket: Socket
+
+  @Synchronized
+  fun setSocket(host: String, port: String) {
+    try {
+      mSocket = IO.socket("$host:$port")
+    } catch (e: URISyntaxException) {}
+  }
+
+  @Synchronized
+  fun getSocket(): Socket {
+    return mSocket
+  }
```

Comment:
you can use a property getter and setter instead: https://kotlinlang.org/docs/properties.html#getters-and-setters